py/selenium/webdriver/support/expected_conditions.py

```diff
@@ -101,6 +101,22 @@ def __init__(self, locator):
 
     def __call__(self, driver):
         return _find_elements(driver, self.locator)
+        
+class visibility_of_all_elements_located(object):
+    """ An expectation for checking that there is at least one element visible
+    on a web page.
+    locator is used to find the element
+    returns the list of WebElements once they are located
+    """
+    def __init__(self, locator):
+        self.locator = locator
+
+    def __call__(self, driver):
+        visible_elements = []
+        for element in _find_elements(driver, self.locator):
+            if _element_if_visible(element) is not False:
```

Comment:
I think this line would be better written as:
`if not _element_if_visible(element):`